### DISCUSSION TITLE
Support for Ring Intercom?

### DISCUSSION THREAD

**Discussion by mensa84** [dgreif/ring#1082] [General]
Hi,

does or will this plugin also support the new Ring Intercom?

**Comment by fluxbeard** [comment]
I would love to have support for the Intercom!!

**Comment by knikkabokka** [comment]
I concur with this question: Is support planned for the all new "Ring Intercom"? Would be awesome!

I would love to see the following functions: 

- A switch that activates the door opener
- An automation switch that can trigger actions when someone rings the doorbell. 
- An automation switch that can trigger actions when the door opener is pressed.

And the following functions would be absolutely unbelievably great, the icing on the cake, so to speak: 

- Intercom function via the HomePod or HomePod Mini. So I don't always have to get the phone out to talk to the person at the door 
- The doorbell is additionally played via the HomePod or HomePod Mini (at best even with selectable ringtones)
- A function that automatically activates the door opener when my iPhone logs back into my WLAN after a period of absence and the doorbell is pressed shortly afterwards (so that when I arrive home, I only have to press my own doorbell button briefly to get into the stairwell)

(But I have no idea if this would even be technically possible with HomeKit and / or Homebridge ...).

**Comment by dgreif** [comment]
This is being tracked in #1083
